packages/components/src/components/domainproperties/list/models.spec.ts

```diff
@@ -1,10 +1,51 @@
 import {ListModel} from "./models";
-
+import {DEFAULT_LIST_SETTINGS} from "../../../test/data/constants";
+import getDomainDetailsJSON from "../../../test/data/property-getDomainDetails.json";
 
 describe('ListModel', () => {
-    test("", () => {
-        const model = ListModel.create({
 
-        })
+    test("new model", () => {
+        const newModel = ListModel.create(null, DEFAULT_LIST_SETTINGS);
+
+        expect(newModel.getIn(['domain', 'name'])).toEqual(undefined);
+        expect(newModel.getIn(['domain', 'domainId'])).toEqual(null);
+        expect(newModel.getIn(['name'])).toEqual(null);
+        expect(newModel.getIn(['keyType'])).toEqual(null);
+
+        expect(newModel.isNew()).toEqual(true);
```

Comment:
the jest tests have a helper for `toBeTruthy()`, so I think you can do something like:
`expect(newModel.isNew()).toBeTruthy();`